e2e/Uno/HelloUnoWorld.Shared/App.xaml.cs

```diff
@@ -0,0 +1,129 @@
+﻿using HelloUnoWorld.Views;
+using HelloUnoWorld.Dialogs;
+using Microsoft.Extensions.Logging;
+using Prism;
+using Prism.DryIoc;
+using Prism.Ioc;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+namespace HelloUnoWorld
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : PrismApplication
+    {
+        /// <summary>
+        /// Initializes the singleton application object.  This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            ConfigureFilters(global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory);
+
+            this.InitializeComponent();
+            this.Suspending += OnSuspending;
```

Comment:
if this is something that is going to be in every app would this perhaps be better to just provide an protected virtual implementation for in PrismApplication?

Reply:
If could be placed in the application base, but devs may not want to get the dependency to all loggers (Debug, Console) as well.

Reply:
sorry I just realized this seemed to have grabbed several lines... this was more in reference to the `this.Suspending += OnSuspending`

Reply:
I see. Yes, it can be placed as a virtual method.

Reply:
It's not a virtual method called `OnSuspending`